src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@ public class TorpedoStore {
 
   private int torpedoCount = 0;
 
+  Random generator = new Random();
```

Comment:
Így valóban hasznosabb, kérlek kommenteld mellé, hogy hol fogod használni, hogy átláthatóbb legyen